src/mui/input/NumberInput.spec.js

```diff
@@ -9,11 +9,18 @@ describe('<NumberInput />', () => {
     const defaultProps = {
         source: 'foo',
         meta: {},
-        input: {},
+        input: {
+            onBlur: sinon.spy(),
+            onChange: sinon.spy(),
+        },
     };
 
     it('should use a mui TextField', () => {
-        const wrapper = shallow(<NumberInput {...defaultProps} input={{ value: 'hello' }} />);
+        const props = { ...defaultProps };
```

Comment:
I don't understand this change, since you don't need to test the `onBlur` or `onChange`, you don't need to set them in the `input`.